src/mqueryfront/src/query/QueryMatches.js

```diff
@@ -58,6 +92,12 @@ const QueryMatches = (props) => {
                                     itemList={downloadDropdownList}
                                 />
                             </span>
+                            {filters.length > 0 && (
+                                <span className="border rounded p-1 pull-right">
+                                    <FilterIcon color="grey" />
+                                    {filtersHead}
+                                </span>
```

Comment:
@psrok1 @KWMORALE @JaroPowerH why are FilterIcon and ActionRemove are forced for Black color? This way, setting text-color on span and parent objects will not apply to the icons?

For example, I'd prefer removing `color="grey"` from line 97 in favor of adding `text-secondary` class in line 96. This is currently not possible because FilterIcon's default props set the icon color to black, inlined (see FilterIcon.js), and thus ignoring the color scheme specified by its parent